include/aspect/newton.h

```diff
@@ -61,6 +61,16 @@ namespace aspect
    * A Class which can declare and parse parameters and creates
    * material model outputs for the Newton solver.
    */
+
+  enum class PreconditionerStabilization
+  {
+    SPD, PD, symmetric, none
+  };
+  enum class VelocityBlockStabilization
+  {
+    SPD, PD, symmetric, none
```

Comment:
Thinking about it, since the two are the same, could you not use the same `enum` for both purposes, and simply have two variables of this enum type?